Source/Plugins/Core/com.equella.core/js/tsrc/modules/SearchModule.ts

```diff
@@ -221,18 +366,35 @@ export const searchItems = ({
   return OEQ.Search.search(API_BASE_URL, searchParams);
 };
 
+const getCollectionDetails = async (
+  collectionUuids: string[]
+): Promise<Collection[] | undefined> => {
+  const collectionList = await collectionListSummary([
+    OEQ.Acl.ACL_SEARCH_COLLECTION,
+  ]);
+  const filteredCollectionList = collectionList.filter((c) =>
+    collectionUuids.includes(c.uuid)
+  );
+  return filteredCollectionList.length > 0 ? filteredCollectionList : undefined;
+};
+
+const getUserDetails = async (
+  userId: string
+): Promise<OEQ.UserQuery.UserDetails | undefined> => {
```

Comment:
Similar to above, this is probably more so a `find` rather than a get. But I do also wonder if `undefined` is actually ever returned. :thinking: 
```suggestion
const findUser = async (
  userId: string
): Promise<OEQ.UserQuery.UserDetails | undefined> => {
```